### ISSUE TITLE
Pin the Status Center Open

### ISSUE THREAD

**Issue by o0Lil-Luna0o** [files-community/Files#15431]
### What feature or improvement do you think would benefit Files?

Allowing users to have the option to pin the status center (active file transfers, etc) open so it stays open even if we click elsewhere in the app or click out of the app. 

### Requirements

- Allows users to keep the status center open, maintaining constant view of all active tasks

### Files Version

3.4.1.0

### Windows Version

10.0.22631.3593

**Comment by Josh65-2201** [comment]
Thanks for the feedback, I've added this to the project board.

**Comment by 0xicl33n** [comment]
it would also be nice to be able to popout dialogue boxes with the file operation information like windows does. 

eg:
![Image](https://github.com/files-community/Files/assets/2192160/3046334b-1371-4c71-b424-1b6d8752e139)

**Comment by 0x5bfa** [comment]
@0xicl33n Isn't it annoying that you have to bring the dialog to the foreground when it went to behind?
This is why we implement it in static and fixed place.

**Comment by 0xicl33n** [comment]
> @0xicl33n Isn't it annoying that you have to bring the dialog to the foreground when it went to behind? This is why we implement it in static and fixed place.

i have multiple monitors and like to use my right side monitor for those dialog boxes so i can keep an eye on the file transfer progress. im not saying they should pop out by default, but an option would be nice

**Comment by protoman** [comment]
Hello. My wife HATES with passion the progress ring, both in apps like Google Chrome but also in files. It would be great that the progress could be kept open, so she can look at the copying progress at any time, without having to click in the ring to see progress.
While I do not have such strong feeling about this UI idea, I all for giving more power and control to the user to see progress permanently. 
Thanks for your understanding, I hope this feature is implemented :)

**Comment by abendrothj** [comment]
@yaira2 Did you have a specific design in mind for implementation? Would love to do it to your specifics so that my pr would be merged cleanly. 👍

**Comment by yair100** [comment]
The design I had in mind is similar to the Shelf Pane (currently available under `Settings > Appearance > Toolbars` when building from source) which shows on the right side of the app.

<img width="321" height="539" alt="Image" src="https://github.com/user-attachments/assets/2201771d-c26f-49e2-9a61-5cfc22e64082" />

Some behaviors that I haven't settled on yet:
- What happens if the Shelf Pane is opened when the user pins the Status Center?
- What happens if the user makes the Status Center very wide and then makes the window smaller?

These are important details that need to be considered.

**Comment by Jay-o-Way** [comment]
@yaira2 this might be a weird idea, but, why not use a tab? 😉

**Comment by yair100** [comment]
> [@yaira2](https://github.com/yaira2) this might be a weird idea, but, why not use a tab? 😉

Is this in reference to my question `What happens if the Shelf Pane is opened when the user pins the Status Center?`? This is certainly an option to consider.

**Comment by Jay-o-Way** [comment]
> Is this in reference to my question

Nope. Just a loose thought. Thinking of browsers. Plenty of space and tear-out ability ready to go.

**Comment by mdtauk** [comment]
Shelf and Status could split their pane, if both opened? 

But there is also the preview/details pane to consider too, and what happens when the window is too narrow to accommodate the shelf and or the pinned Status Centre pane?

I think we could add the wording "when space allows" to the settings and we auto-collapse to flyouts when the window is too small.  The File/Folder area needs to always have priority.

**Comment by Jay-o-Way** [comment]
@mdtauk I think a Tab wil have enough space. Also, it can be used next to the shelf pane, which gives easy manipulation. Just like the Download folder would.

But just to check: are the files or events in this window "sticky"?

**Comment by mdtauk** [comment]
> [@mdtauk](https://github.com/mdtauk) I think a Tab wil have enough space. Also, it can be used next to the shelf pane, which gives easy manipulation. Just like the Download folder would.
> 
> But just to check: are the files or events in this window "sticky"?

Using a tab to hold the Status Centre is another valid choice.  But I assume wanting to pin this open, is more about keeping an eye on progress, whilst using the app.  If they have to be within a tab, the only way to show this alongside doing other things is with a split view turned on.

Clicking on a tab to check, is the same as clicking on the button to open the flyout, so is not really a different functionality.

**Comment by yair100** [comment]
> I think we could add the wording "when space allows" to the settings and we auto-collapse to flyouts when the window is too small. The File/Folder area needs to always have priority.

@mdtauk prioritizing the space for the file area and collapsing the pane is reasonable.

**Comment by Jay-o-Way** [comment]
> wanting to pin this open, is more about keeping an eye on progress, whilst using the app. If they have to be within a tab, the only way to show this alongside doing other things is with a split view turned on.

I dunno about that, a tab can be torn out and placed in a corner of a screen, right?

**Comment by o0Lil-Luna0o** [comment]
> Shelf and Status could split their pane, if both opened?
> 
> But there is also the preview/details pane to consider too, and what happens when the window is too narrow to accommodate the shelf and or the pinned Status Centre pane?
> 
> I think we could add the wording "when space allows" to the settings and we auto-collapse to flyouts when the window is too small. The File/Folder area needs to always have priority.

The window could always just expand further to allow the panes to open without compromising the width/viewability of the main files area.

**Comment by mdtauk** [comment]
> The window could always just expand further to allow the panes to open without compromising the width/viewability of the main files area.

Not sure that window resizing is a good idea, as the user should have full control of the window's size and position.  Not respecting that would be bad UX.

> I dunno about that, a tab can be torn out and placed in a corner of a screen, right?

If space is at a premium that the user has chosen to make it small,  having a duplicate window, with all the toolbars, sidebar, etc may not be a good expectation to have.

**Comment by Josh65-2201** [comment]
I agree that tabs should not be used, it too big for the content and not pinning that is asked for.

> What happens if the Shelf Pane is opened when the user pins the Status Center?

It would make sense to have it to the right of the shelf pane 


> What happens if the user makes the Status Center very wide and then makes the window smaller?

Would there be a need to have it resizable? Since text now wraps and there is a lot more vertical room, the width could be fixed at the current status center width.

As for making the window small with both options enabled, I think it would make sense to keep them as is. If the user does enable them both and make Files as small as possible, it would make sense it's not that usable, just like making the info pane maxed width and the shelf open.
